pages/cloud/storage/nas_nfs/guide.en-gb.md

```diff
@@ -1,236 +1,194 @@
 ---
-title: 'Mount your NAS via NFS share'
+title: Mounting HA-NAS via NFS share
 slug: nas-nfs
-excerpt: 'Find out how to mount a NAS over an NFS share here.'
+excerpt: Find out how to connect to your HA-NAS using an NFS share
 section: NAS
 order: 03
 ---
 
-**Last updated 21st February 2022**
-
-## Requirements
-
-This guide allows you to mount NFS on the most common distributions. To mount an NFS share, you must have:
-
-- a [Dedicated Server](https://www.ovhcloud.com/en-gb/bare-metal/) **or** a [VPS](https://www.ovhcloud.com/en-gb/vps/) **or** a [Public Cloud Instance](https://www.ovhcloud.com/en-gb/public-cloud/).
-- a [HA-NAS solution](https://www.ovh.co.uk/nas/).
-- an NFS-compatible distribution.
+**Last updated 16th September 2022**
```

Comment:
Can we generate this string at build?
This looks is a bit tricky and easy to forget to set this the right way.

Reply:
Not for the moment, we have no issues so far writing it manually. Automation is being studied though.